runelite-client/src/main/java/net/runelite/client/plugins/groundmarkers/GroundMarkerPlugin.java

```diff
@@ -278,6 +287,28 @@ else if (option.equals(LABEL))
 		}
 	}
 
+	@Subscribe
+	public void onConfigChanged(ConfigChanged event)
+	{
+		if (event.getGroup().equals(GroundMarkerConfig.GROUND_MARKER_CONFIG_GROUP)
+			&& (event.getKey().equals(GroundMarkerConfig.SHOW_IMPORT_EXPORT_KEY_NAME)
+				|| event.getKey().equals(GroundMarkerConfig.SHOW_CLEAR_KEY_NAME)))
+		{
+			// Have order consistent by removing everything, then adding according to config
+			sharingManager.removeMenuOptions();
+			sharingManager.removeClearOption();
```

Comment:
I suppose these should be consolidated into the existing `removeMenuOptions` method since they're never called separately.

Reply:
I combined the two remove methods together and renamed the two add methods, especially `addMenuOptions()` to better reflect what they do.